gen.treesitter/src/main/java/com/github/gumtreediff/gen/treesitter/AbstractTreeSitterGenerator.java

```diff
@@ -50,6 +51,8 @@ protected TreeContext generate(Reader r) throws IOException {
 
     @Override
     protected String[] getCommandLine(String file) {
-        return new String[]{TREESITTER_CMD, file, getParserName()};
+        return PYTHON_CMD == null
+                ? new String[]{TREESITTER_CMD, file, getParserName()}
+                : new String[]{PYTHON_CMD, TREESITTER_CMD, file, getParserName(), getParserName()};
```

Comment:
I am not sure why this one needs two `getParserName()` ?

Reply:
Sorry, this is a problem due to my carelessness😨